src/data/faq_de.json

```diff
@@ -87,6 +87,125 @@
+                        "active": true,
+                        "textblock": [
+                            "Ein negatives Schnelltest-Ergebnis ist nach der Übermittlung für 48 Stunden gültig und kann - sofern gesetzlich festgelegt - als Nachweis verwendet werden. Wenn Sie auf dem Hauptbildschirm der App auf Ihr negatives Testergebnis tippen, sehen Sie einen Zähler, welcher Ihnen zeigt, wie lange Ihr Testergebnis bereits zur Verfügung steht. Nach 48 Stunden wird Ihr Schnelltest-Ergebnis automatisch entfernt. Anschließend können Sie einen weiteren Schnelltest machen und in der App registrieren.",
+                            "Für positive Schnelltest-Ergebnisse gibt es keine begrenzte Gültigkeit, deshalb wird Ihnen in diesem Fall kein Zähler angezeigt. Sobald Sie ein positives Testergebnis erhalten, befolgen Sie umgehend die Anweisungen, die Ihnen in der App angezeigt werden. "
+                        ]
+                    },
+                    {
+                        "title": "Welche Privilegien leiten sich von einem negativen Schnelltest-Ergebnis ab? Kann ich mein Testergebnis in der App als Nachweis für einen Einlass in Geschäfte oder Besuch von Friseursalons, Freizeitaktivitäten, etc. verwenden?",
+                        "anchor": "rat_privileges",
+                        "active": true,
+                        "textblock": [
+                            "Ein negatives Testergebnis kann Ihnen, falls gesetzlich festgelegt, als Nachweis für das Vorliegen eines negativen Schnelltest-Ergebnisses dienen. Die Anerkennung von Test-Nachweisen kann allerdings von Bundesland zu Bundesland variieren. Sie sollten sich deshalb über die jeweiligen Kriterien in Ihrem Bundesland informieren."
+                        ]
+                    },
+                    {
+                        "title": "Wie handele ich, wenn mein Schnelltest-Ergebnis positiv ist?",
+                        "anchor": "rat_what_if_positive_result",
+                        "active": true,
+                        "textblock": [
+                            "In Fall eines positiven Testergebnisses sollten Sie Ihr Testergebnis teilen und umgehend einen PCR-Test machen, um das Ergebnis zu bestätigen oder zu widerlegen. Bei einem positiven PCR-Test sollten Sie das Testergebnis erneut teilen, um Kontakte zu warnen, die Sie möglicherweise zwischen den beiden Tests getroffen haben. Voraussetzung dabei ist, dass die Kontaktpersonen ebenfalls die Corona-Warn-App nutzen und die Risikoberechnung ein hohes Risiko bestimmt, basierend auf Dauer und Abstand des Kontakts. Sollte der PCR-Test negativ sein, haben Sie nicht die Möglichkeit, Ihren positiven Schnelltest und die Warnung zurückzuziehen."
+                        ]
+                    },
+                    {
+                        "title": "Wenn ich eine hohe Risikobegegnung in der App angezeigt bekomme, d.h. eine rote Karte/Kachel, sollte ich dann einen Schnelltest oder PCR-Test machen?",

```

Comment:
This does not really belong in this section IHMO.

Reply:
I think I see what you mean. The problem is, that an FAQ article could be positioned at several places in the FAQ. In this case it is because of the keyword Schnelltest here in this block.

I suggest to keep it here, since we think that people with look at the whole **all** new FAQ entries in this new Schnelltest section. Afterwards we can change the logic of grouping articles. What do you think?

Reply:
Okay, agreed!